src/WebJobs.Script/runtimeassemblies.json

```diff
@@ -67,6 +67,14 @@
       "name": "Grpc.Net.Common",
       "resolutionPolicy": "private"
     },
+    {
```

Comment:
Should this be minorMatchOrLower? We may have customers using healthcheck in InProc.

Reply:
They may be using health checks, but we don't need to interop these 2 assemblies with customer code. So we will keep it `private` so they get their own copy.